README.md

```diff
@@ -1,24 +1,24 @@
-# README
+# Contact.me
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Keep track of your contacts, all in one place, anytime, anywhere!
 
-Things you may want to cover:
+## Technologies
 
-* Ruby version
+- Ruby on Rails
+- React.js
+- Materialize CSS
```

Comment:
Really? Materialize? What's you thoughts on this, why this choice?

Reply:
I would just take it out for now till we figure what is best